package.json

```diff
@@ -26,7 +26,7 @@
     "jest": "^26.4.0",
     "jest-emotion": "^10.0.32",
     "jest-localstorage-mock": "^2.4.3",
-    "lerna": "^3.22.1",
+    "lerna": "^5.0.0",
```

Comment:
Do we need this to be `5.0.0` _exactly_? I can't recall if that was just the minimum version or not.  if not: could we update to the latest v5 version? Latest I see is `5.6.2`, this will have the most up to date support 💯

Reply:
Ah, good point. I'll try that really quick and see if that still behaves the same.